server/utils/pluginapi/api.go

```diff
@@ -26,14 +26,19 @@ func (a *API) GetMattermostUserStatus(mattermostUserID string) (*model.Status, e
 }
 
 func (a *API) GetMattermostUserStatusesByIds(mattermostUserIDs []string) ([]*model.Status, error) {
-	return a.api.GetUserStatusesByIds(mattermostUserIDs)
+	st, err := a.api.GetUserStatusesByIds(mattermostUserIDs)
+	if err != nil {
+		return nil, err
```

Comment:
@levb If this function is implemented as just `return a.api.GetUserStatusesByIds(mattermostUserIDs)`, a nil error will be treated as a non-nil value to the caller.

You and I talked once about handling `model.AppError` and `error`, specifically that an interface pointer to a (nil pointer to a concrete type) is not a nil interface pointer, but I don't understand why this needs to be here in this case.